nixos/tailscaled-module.nix

```diff
@@ -0,0 +1,215 @@
+self: {
```

Comment:
What's the copyright on this? Add the header?

Reply:
yeah, header on both of those files.  Looks like that's already been done 👍🏻

Reply:
oh, sorry... I glanced at this too fast.  I thought we were just talking mechanics of whether these files should have headers.

Where did this file actually come from?